### PR TITLE
Connect Postgres DB to Back-end

### PR DESCRIPTION
**Pull request by YangengChen**

### CODE REVIEW

File: client/build/asset-manifest.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "runtime~main.js": "/static/js/runtime~main.a8a9905a.js",
+    "runtime~main.js.map": "/static/js/runtime~main.a8a9905a.js.map",
+    "static/js/2.8cc23b2b.chunk.js": "/static/js/2.8cc23b2b.chunk.js",
+    "static/js/2.8cc23b2b.chunk.js.map": "/static/js/2.8cc23b2b.chunk.js.map",
+    "index.html": "/index.html",
+    "precache-manifest.1e82b00c90a7d9259853f3ecde559d1b.js": "/precache-manifest.1e82b00c90a7d9259853f3ecde559d1b.js",
+    "service-worker.js": "/service-worker.js",
+    "static/css/main.ba49ec77.chunk.css.map": "/static/css/main.ba49ec77.chunk.css.map"
+  }
+}
```
Comment (skassam21) [inline client/build/asset-manifest.json:15]: I feel like we should gitignore the client/build folder

File: server/app.py [hunk 2]
```diff
@@ -1,10 +1,24 @@
 from flask import Flask
 from api.ping_handler import ping_handler
 from api.home_handler import home_handler
-
+from flask_sqlalchemy import SQLAlchemy
 
 app = Flask(__name__)
 
+#creates a postgres database connection with postgresql://username:password@localhost/dbname
+app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:1234@localhost/team-winter'
```
Comment (skassam21) [inline server/app.py]: can we make this a variable in .env and then gitignore the .env file

Comment (skassam21) [reply server/app.py]: If you are going to do this, you should write a quick readme on how to start the app with postgres

File: server/app.py [hunk 3]
```diff
@@ -1,10 +1,24 @@
...
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+
+db = SQLAlchemy(app)
+
+#test database, remove as necessary.
+class Test(db.Model):
+    __tablename__ = 'Tests'
+    id = db.Column(db.Integer, primary_key = True)
```
Comment (skassam21) [inline server/app.py]: This should be in a separate folder structure, all models should have their own folder structure

**Comment by skassam21** [comment]
You will have to think about how to handle migrations as well in this app

File: server/app.py (hunk 3 above)
Comment (YangengChen) [reply server/app.py]: Yes, I was just testing it real quick!

**Review by skassam21 — APPROVED** [review]

**Pull request merged**